### PR TITLE
Revert "feat:add  faculty year ticket"

### PR DESCRIPTION
**Pull request by new5558**
Reverts hu-tao-supremacy/frontend#64

### CODE REVIEW

**Review by boompttn — APPROVED** [review]
LGTM

File: src/modules/eventRegister/personal-info-des/PersonalInfoDes.vue [hunk 1]
```diff
@@ -39,15 +39,15 @@
...
+          :value="user.chulaId"
           class="w-full h-4 disabled: bg-white text-primary-6"
           disabled
         />
       </div>
       <div class="flex flex-col w-1/3">
         <div>Year</div>
         <BaseTextInput
-          :value="user.academicYear"
+          :value="user.chulaId"
```
Comment (poom-kitti) [inline src/modules/eventRegister/personal-info-des/PersonalInfoDes.vue:50]: The hard code value will break now

File: src/modules/eventRegister/personal-info/PersonalInfo.vue [hunk 2]
```diff
@@ -38,7 +38,7 @@
       <div class="flex flex-col w-1/3 mr-4">
         <div>Faculty</div>
         <BaseTextInput
-          :value="faculty"
+          :value="user.chulaId"
```
Comment (poom-kitti) [inline src/modules/eventRegister/personal-info/PersonalInfo.vue:41]: Hard code value will break

**Review by poom-kitti — APPROVED** [review]
LGTM after change

**Pull request merged**